modules/lmdbbackend/lmdbbackend.hh

```diff
@@ -51,7 +51,7 @@ public:
   bool replaceRRSet(uint32_t domain_id, const DNSName& qname, const QType& qt, const vector<DNSResourceRecord>& rrset) override;
 
   void getAllDomains(vector<DomainInfo> *domains, bool include_disabled=false) override;
-  void lookup(const QType &type, const DNSName &qdomain, int zoneId, DNSPacket *p=nullptr) override;
+  void lookup(const QType &type, const DNSName &qdomain, int zone_id, DNSPacket *p) override;
```

Comment:
why is `nullptr` gone? (I agree that it currently changes nothing, but it's a good failsafe.)

Reply:
Ah, might be a merge booboo, will check.